CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 * Fixed an issue where the **validate** command failed to validate the release notes of beta integrations.
 * Updated the **upload** command to support indicator fields.
+* Added a validation that some specific fields in the yml file is not changed from true to false, or totally removed
```

Comment:
```suggestion
* Added a validation that integration properties are not removed (feed, mappers, mirroring).
```

Reply:
`* Added a validation that integration properties (such as feed, mappers, mirroring, etc) are not removed.`